scss/_buttons.scss

```diff
@@ -83,8 +83,10 @@
   &:first-child:active,
   &.active,
   &.show {
-    color: var(--#{$prefix}btn-active-color);
-    background-color: var(--#{$prefix}btn-active-bg);
+    // WILL REMOVING THESE CAUSE OTHER PROBLEMS?
+    // color: var(--#{$prefix}btn-active-color);
+    // background-color: var(--#{$prefix}btn-active-bg);
+
     // Remove CSS gradients if they're enabled
     background-image: if($enable-gradients, none, null);
     border-color: var(--#{$prefix}btn-active-border-color);
```

Comment:
Should we move this one out too? It also seems to be an issue for non button variants e.g. `.btn .my-custom-border-override` case as well. You can see from the chrome devtools screen grab below.